samza-core/src/main/java/org/apache/samza/coordinator/MetadataResourceUtil.java

```diff
@@ -62,7 +59,13 @@ public void createResources() {
     createChangelogStreams();
   }
 
+  @VisibleForTesting
   void createChangelogStreams() {
-    ChangelogStreamManager.createChangelogStreams(jobModel.getConfig(), jobModel.maxChangeLogStreamPartitions);
+    ChangelogStreamManager.createChangelogStreams(config, jobModel.maxChangeLogStreamPartitions);
```

Comment:
Would be great if we can log a message at appropriate level that we're creating the change-log and checkpoint stream. Currently it is not being logged anywhere, so it'll be useful during debugging.

Reply:
They are being logged in the create calls themselves. That's how I was able to figure out these weren't being called.